docs/config.md

```diff
@@ -704,9 +704,9 @@ Below is a list of the most common public gateway setups.
    ```
    **Note I:** this enables automatic redirects from content paths to subdomains:  
    `http://dweb.link/ipfs/{cid}` → `http://{cid}.ipfs.dweb.link`  
```

Comment:
The formatting here is no longer consistent. What about just making this "Note:" instead of "Note I:"?